src/features/event-manage/event-create/hooks/usePresignedUrlHook.ts

```
@@ -0,0 +1,52 @@
+
+    return response.data.result?.preSignedUrl;
+  } catch (error) {
+    console.error('Presigned URL 요청 실패:', error);
+    throw error;
+  }
+};
+
+export const putS3Image = async ({ url, file }: { url: string; file: File }) => {
+  try {
+    delete axiosClient.defaults.headers.common.Authorization;
+    console.log('업로드할 URL:', url);
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)